client/src/js/hmm/components/Detail.js

```diff
@@ -6,6 +6,7 @@ import styled from "styled-components";
 import { Badge, BoxGroup, BoxGroupHeader, Label, LoadingPlaceholder, NotFound, Table, ViewHeader } from "../../base";
 import { getHmm } from "../actions";
 import { HMMTaxonomy } from "./Taxonomy";
+import { device } from "../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['../../base' import is duplicated. (no-duplicate-imports)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4687100)